cmd/nodecmd/wiz.go

```diff
@@ -172,7 +172,9 @@ func wiz(cmd *cobra.Command, args []string) error {
 
 	if !clusterAlreadyExists {
 		globalNetworkFlags.UseDevnet = true
-		useAvalanchegoVersionFromSubnet = subnetName
+		if len(useCustomAvalanchegoVersion) == 0 {
```

Comment:
also check for useLatestAvalanchegoReleaseVersion, useLatestAvalanchegoPreReleaseVersion